tools/test-case-generators/generate-all-test-cases

```diff
@@ -1289,6 +1317,14 @@ def get_args():
             help="Don't delete the workdir created on the remote host after finishing.",
             default=False
         )
+    parser.add_argument(
+            "--repofrompath",
```

Comment:
Why `repofrompath` and not something like `rpmrepo` or `extrarepo`?  I feel like the `frompath` part implies that the value should be a filesystem path.

Reply:
From `man dnf`:
```
 --repofrompath <repo>,<path/url>
              Specify a repository to add to the repositories for this query.  This option can be used multiple times.
```